.github/workflows/gem-push.yml

```diff
@@ -3,6 +3,11 @@ name: scc-codestyle
 on:
   push:
     branches: [ master ]
+    paths:
+      - default.yml
+      - Gemfile.yml
+      - Gemfile.lock
```

Comment:
Instead of Gemfile.yml (typo?) and Gemfile.lock (not in git), maybe it's better to add [scc-codestyle.gemspec](https://github.com/SUSE/scc-codestyle/blob/master/scc-codestyle.gemspec) here?